apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/MapExpectationSamples.kt

```diff
@@ -0,0 +1,246 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class MapExpectationSamples {
+
+    @Test
+    fun toContainBuilder() {
+        expect(mapOf(1 to "a")) toContain (1 to "a")
+
+        fails { // because the map does not contain key 1 with value "b"
+            expect(mapOf(1 to "a")) toContain (1 to "b")
+        }
+    }
+
+    @Test
+    fun toContainPair() {
+        expect(mapOf(1 to "a")) toContain (1 to "a")
+
+        fails { // because the map does not contain key 1 with value "b"
+            expect(mapOf(1 to "a")) toContain (1 to "b")
+        }
+    }
+
+    @Test
+    fun toContainOnlyPair() {
+        expect(mapOf(1 to "a")) toContainOnly (1 to "a")
+
+        fails { // because the map contains key 2 with value "b" in addition
+            expect(mapOf(1 to "a", 2 to "b")) toContainOnly (1 to "a")
+        }
+    }
+
+    @Test
+    fun toContainKeyValue() {
+        // subject inside this block is of type String (actually "a")
+        expect(mapOf(1 to "a")) toContain keyValues(keyValue(1) { this toEqual "a" })
```

Comment:
I would reformat this so that is more obvious which block we mean. For instance like the following 
```suggestion
        expect(mapOf(1 to "a")) toContain keyValues(
          keyValue(1) { // subject inside this block is of type String (actually "a")
            this toEqual "a" 
          }
        )
```
Or like this
```suggestion
        expect(mapOf(1 to "a")) toContain keyValues(keyValue(1) { this toEqual "a" })
        //                                                       | subject inside this block is of type String (actually "a")
```

Reply:
@kacmacuna this one is still open. Same applies to the other samples further below. You did a good job in MapEntryExpectionSamples. Would be nice if you could do the same here as well.

Reply:
Same here I went with first option and reformated it like this:
![Screenshot_42](https://user-images.githubusercontent.com/39156859/120605671-61ff7180-c45f-11eb-856d-31dec90402dd.png)

Reply:
Looks like it. I guess a bug in githubs new beta feature conversation